specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2023-06-01-preview/SourceControls.json

```diff
@@ -213,18 +222,20 @@
             }
           }
         }
-      },
-      "put": {
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.OperationalInsights/workspaces/{workspaceName}/providers/Microsoft.SecurityInsights/sourcecontrols/{sourceControlId}/delete": {
```

Comment:
There is no body in request payload now, as I can see. Should this be Delete operation now or is there any reason for it to be a POST Action?

Reply:
Good catch, thanks. Added definition of the body.